charts/athens-proxy/templates/_helpers.tpl

```diff
@@ -15,6 +15,40 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
+{{/*
+Common labels used by all resources in their meta fields.
+Includes existing labels for passivity, new standard labels, and user-defined extra labels.
+https://helm.sh/docs/chart_best_practices/labels/
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
+*/}}
+{{- define "athens.metaLabels" -}}
+{{- /* Allow an app suffix name to be passed in to append to the fullname */}}
+{{- $defaultAppName := include "fullname" . }}
+{{- $appName := printf "%s%s" $defaultAppName (default "" .appSuffix) }}
+{{- /* Existing Legacy labels for passivity */}}
```

Comment:
I added these comments with the intention to document why they were here, however this top one does create an extra empty line the in the helm template

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: release-name-athens-proxy-jaeger
  labels:
    
    app: release-name-athens-proxy-jaeger
    chart: athens-proxy-0.7.0
    release: "release-name"
    heritage: "Helm"
    app.kubernetes.io/name: release-name-athens-proxy-jaeger
    helm.sh/chart: athens-proxy-0.7.0
    app.kubernetes.io/managed-by: "Helm"
    app.kubernetes.io/instance: "release-name"
    app.kubernetes.io/version: "v0.12.0"
    athensIs: awesome
spec:
```

This output seems generally accepted by the helm docs https://helm.sh/docs/chart_best_practices/templates/#whitespace-in-generated-templates but i'm happy to remove these comments if the tradeoff isn't worth it

Reply:
As long as the indentation is correct, it's fine. I've seen many helm charts with some empty lines. The comment should generally not be necessary when the code is readable, but it may help newcomers and does not hurt.